### ISSUE TITLE
Database screen missing race labels

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#9]
```
Some race labels are missing from the database screen.

Fixed in v0.90

```

Original issue reported on code.google.com by `akarn...@gmail.com` on 11 Apr 2011 at 9:00

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 11 Apr 2011 at 9:01
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.90
```

Original comment by `akarn...@gmail.com` on 15 Apr 2011 at 9:14
- Changed state: **Fixed**
